plugins/thunar-apr/thunar-apr-image-page.c

```diff
@@ -173,7 +172,9 @@ thunar_apr_image_page_init (ThunarAprImagePage *image_page)
   /* add labels for the Exif info */
   for (n = 0; n < G_N_ELEMENTS (TAIP_EXIF); ++n)
     {
-      label = gtk_label_new (_(TAIP_EXIF[n].name));
+      const gchar *exif_title = exif_tag_get_title ( TAIP_EXIF[n] );
```

Comment:
Ad 1. If you want I could try to ask libexif maintainers about that.
Ad 2. I believe it's better for entire open source community to improve libexif and it's always less translation to maintain for thunar maintainers .. but this is more political issue ;-) Maybe we can ask some XFCE guru what he prefer .. or maybe you are such guru , then it's your decision ;-)